File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_66106.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify


def BenchmarkTest66106():
    multipart_value = request.form.get('multipart_field', '')
    data = f'{multipart_value:.200s}'
    int(str(data))
    return jsonify({"result": "success"})

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest66106.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest66106 {

    @GetMapping("/BenchmarkTest66106")
    public void BenchmarkTest66106(HttpServletRequest request, HttpServletResponse response) throws Exception {
        String secretValue = "p4ssw0rd_test_xyz";
        String data = "" + secretValue;
        try (java.sql.Connection authConn = java.sql.DriverManager.getConnection(
                "jdbc:postgresql://db.svc.local/app", "appuser", data)) {
            response.getWriter().print("{\"auth\":\"ok\"}");
        }
    }
}
